client/extensions/woocommerce/app/reviews/review-replies.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External depedencies
+ */
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { fetchReviewReplies } from 'woocommerce/state/sites/review-replies/actions';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import { getReviewReplies } from 'woocommerce/state/sites/review-replies/selectors';
+import ReviewReply from './review-reply';
+
+class ReviewReplies extends Component {
+	static propTypes = {
+		siteId: PropTypes.number,
+		review: PropTypes.shape( {
+			status: PropTypes.string,
+			name: PropTypes.string,
+			id: PropTypes.number,
+		} ).isRequired,
+	};
+
+	componentDidMount() {
+		const { siteId, review } = this.props;
+		if ( siteId ) {
+			this.props.fetchReviewReplies( siteId, review.id );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		const { siteId, review } = this.props;
+		const newSiteId = newProps.siteId || null;
+		const oldSiteId = siteId || null;
+		if ( oldSiteId !== newSiteId ) {
+			this.props.fetchReviewReplies( newSiteId, review.id );
+		}
+	}
+
+	renderReply = ( replyId, i ) => {
+		const { review } = this.props;
+		return <ReviewReply
+			key={ i }
+			reviewId={ review.id }
+			replyId={ replyId }
+		/>;
+	}
+
+	render() {
+		const { replyIds, review, translate } = this.props;
+		const repliesOutput = replyIds.length && replyIds.map( this.renderReply ) || null;
+
+		const textAreaPlaceholder = translate( 'Reply to %(reviewAuthor)s…', {
+			args: { reviewAuthor: review.name }
+		} );
+		const submitButton = 'approved' === review.status ? translate( 'Send reply' ) : translate( 'Approve and send reply' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **96** times:
`translate( 'Approve and Reply' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).